atlp-rwanda/code-ninjas-fe
# ft-role-setting

### What does this PR do?

User Role Settings UI

### Description of Task to be completed?

Different user roles are supported within the application with each role allowed different level of access


### How should this be manually tested?
Get started By Running commands bellow:
  - clone repo
  - switch to [ft-role-setting-181414722](https://github.com/atlp-rwanda/code-ninjas-fe/tree/ft-role-setting-181414722)
  - branch
  - Run `yarn install`: to install all dependencies
  - Run `yarn dev`: to start the application
  - Open your browser & and login. if you login us requester you will not see user link in the menu. if you are super admin you will see it. 
  - After logging in us super admin click users link and follow the instructions to assign role to other users

`

Routes:


`



[/dashboard/users](https)
Forgot password PAGE  `super admin`



`

### Any background context you want to provide?
### What are the relevant pivotal tracker stories?
[#181414722](https://www.pivotaltracker.com/story/show/181414722)
### Screenshots (if appropriate)
### Questions:
